### ISSUE TITLE
Timer starts before clicking "start" after closing the laptop during a meditation session. 

### ISSUE THREAD

**Issue by choipaul92** [EmotionCognitionLab/lipbelly#8]
8/31/23 2:36PM

After 10 minutes of meditation, I closed the laptop for 1 minute and 30 seconds. After opening the laptop, the app gave me the message "The session has ended because it has been a long time since...". **But the 20 minute timer started BEFORE i pressed "start."** Please note that I never took my ear sensor off my earlobe during the meditation testing phase.  After I clicked "Start" the message "waiting for pulse signal" appeared and the timer continued (DID NOT RESET TO 20 minutes). Completed the 20 minute meditation session.

**Comment by noah10** [comment]
So I can replicate this, you left your ear sensor on your earlobe during the 1m30s that you had the laptop lid closed?

**Comment by choipaul92** [comment]
Yes

**Comment by noah10** [comment]
I was unable to reproduce this. When I opened the laptop I noticed a few things:

- The timer had stopped when I closed the lid and started when I reopened it. This is expected, as I have the laptop configured to sleep when I closed the lid.
- I received the "no or poor signal" error shortly after opening the lid, not the "The session has ended" message. 
- After waiting a bit, the "no or poor signal" error changed to the "The session has ended" message and the timer reset to 20 minutes.

Are you able to reproduce this?

**Comment by choipaul92** [comment]
I tested this during a 15 minute meditation session and when I opened the lid (after closing it), the timer resumed from where I left off and kept going for a few seconds. Then I got the message "no or poor signal detected" and the timer stopped (giving me the "the session has ended" message). The timer reset to 20 minutes and this time around the timer did not start UNTIL I pressed the "start" button.
